packages/ai/src/chains/operations/operations/edit-styles.ts

```diff
@@ -0,0 +1,39 @@
+import { z } from "zod";
+import { EmbedTemplateStyleDecl } from "@webstudio-is/react-sdk";
+import { parseTailwindToWebstudio } from "@webstudio-is/css-data";
+
+const wsId = z
+  .string()
+  .describe("The element's ws-id that must get the new styles");
+
+export const aiOperation = z.object({
+  operation: z.literal("editStylesWithTailwindCSS"),
```

Comment:
nit: editWithTailwind seems clear and shorter

Reply:
oh I see there is a consistency with `generateInstanceWithTailwindStyles`

in this case it could be editStylesWithTailwind, generateInstanceWithTailwind